### DISCUSSION TITLE
When I reboot my container Transmission populates a ton of old downloads

### DISCUSSION THREAD

**Discussion by hedbert1** [haugene/docker-transmission-openvpn#1963] [Q&A] [answered]
Greetings-
I have searched on this several times, forgive me if I missed a post somewhere.  Whenever I reboot my container and it comes back up, transmission is populated is a bunch of old downloads that I have already removed from the GUI.  Is there a way to clean the database and clean these out?  This is annoying as I need to stop and remove them but avoid the ones that are legit still downloading.  Thank you in advance for the assistance.

**Comment by pkishino** [comment, marked as answer]
This has been brought up a bunch already, mostly linking to #1937 .
You need to stop container, clean your transmission-data folder from resume/cache files then restart

**Reply by hedbert1** [reply]
Greetings-
Thank you.  I did not realize that this is the same issue.  I am sorry for this duplicate post.
